patches/minecraft/net/minecraft/world/level/material/Fluid.java.patch

```diff
@@ -35,6 +42,31 @@
 +         forgeFluidAttributes = createAttributes();
 +      return forgeFluidAttributes;
 +   }
- 
++
     public Optional<SoundEvent> m_142520_() {
        return Optional.empty();
++   }
++
++   private Object renderProperties;
++
++   /*
++      DO NOT CALL, IT WILL DISAPPEAR IN THE FUTURE
++      Call RenderProperties.get instead
++    */
++   public Object getRenderPropertiesInternal() {
++      return renderProperties;
++   }
++
++   private void initClient() {
++      if (net.minecraftforge.fml.loading.FMLEnvironment.dist == net.minecraftforge.api.distmarker.Dist.CLIENT) {
```

Comment:
Should be updated to match https://github.com/MinecraftForge/MinecraftForge/commit/347a0b3c0dc7873b9bf6910f6ef9b99a2d786668